lib/ansible/galaxy/token.py

```diff
@@ -35,6 +48,93 @@
 
 display = Display()
 
+AH_TOKEN_EXPIRES_N_MIN_EARLY = 1
+AH_AUTH_URL = "https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token"
+AH_JWKS_URI = "https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/certs"
+
+# https://www.rfc-editor.org/rfc/rfc7518#section-3.1
+REQUIRED_JWT_HEADER_KEYS = set(['alg', 'typ', 'kid'])
```

Comment:
Unless this module needs to be compatible with Python 2.6, we should really just use native syntax:
```suggestion
REQUIRED_JWT_HEADER_KEYS = {'alg', 'typ', 'kid'}
```